tools/performance/engine-benchmarks/bench_download.py

```diff
@@ -78,57 +80,71 @@
     import numpy as np
     import jinja2
 except ModuleNotFoundError as err:
-    print("ERROR: One of pandas, numpy, or jinja2 packages not installed", file=sys.stderr)
+    print("ERROR: One of pandas, numpy, or jinja2 packages not installed",
+          file=sys.stderr)
     print("Install either with `pip install pandas numpy jinja2` or "
-          "with `apt-get install python3-pandas python3-numpy python3-jinja2`", file=sys.stderr)
+          "with `apt-get install python3-pandas python3-numpy python3-jinja2`",
+          file=sys.stderr)
     exit(1)
 
-CSV_FIELDNAMES = [
-    "label",
-    "score",
-    "commit_id",
-    "commit_author",
-    "commit_timestamp",
-    "bench_run_url",
-    "bench_run_event"
-]
 
+@dataclass
+class CsvRow:
+    label: str
+    score: str
+    commit_id: str
+    commit_title: str
+    commit_timestamp: str
+    commit_author: str
+    bench_run_id: str
+    bench_run_url: str
+    bench_run_event: str
 
-def write_bench_reports_to_csv(bench_reports: List[JobReport], csv_fname: str) -> None:
+
+def write_bench_reports_to_csv(bench_reports: List[JobReport],
+    csv_fname: str) -> None:
     logging.info(
         f"Writing {len(bench_reports)} benchmark reports to {csv_fname}")
+    csv_fieldnames = CsvRow.__annotations__.keys()
     assert len(bench_reports) > 0
-    if not path.exists(path.dirname(csv_fname)):
-        logging.debug(f"Creating directory {path.dirname(csv_fname)}")
-        os.mkdir(path.dirname(csv_fname))
     with open(csv_fname, "w") as csv_file:
-        csv_writer = DictWriter(csv_file, CSV_FIELDNAMES)
+        csv_writer = DictWriter(csv_file, csv_fieldnames)
         csv_writer.writeheader()
         for bench_report in bench_reports:
             for label, score in bench_report.label_score_dict.items():
-                csv_writer.writerow({
-                    "label": label,
-                    "score": score,
-                    "commit_id": bench_report.bench_run.head_commit.id,
-                    "commit_author": bench_report.bench_run.head_commit.author.name,
-                    "commit_timestamp": bench_report.bench_run.head_commit.timestamp,
-                    "bench_run_url": bench_report.bench_run.html_url,
-                    "bench_run_event": bench_report.bench_run.event
-                })
+                commit_title = \
+                  bench_report.bench_run.head_commit.message.splitlines()[0]
+                commit_title = commit_title.replace(",", " ")
+                # Ensure that score is not printed with exponential notation,
+                # Enso cannot easily parse that by default now.
```

Comment:
Let's not forget to address the problem in Enso though :)

Reply:
Sure. Tracked in https://github.com/enso-org/enso/issues/9418